package.json

```diff
@@ -121,7 +121,7 @@
 		"pluralize": "^8.0.0",
 		"prop-types": "^15.7.2",
 		"querystringify": "^2.1.1",
-		"react-final-form": "^4.0.2",
+		"react-final-form": "^6.3.0",
```

Comment:
oh wow... didn't know it was so far behind!!!

Reply:
Yes, there are a few hooks in the latest version.